config/deploy/deployment.yaml.tpl

```diff
@@ -46,6 +46,25 @@ spec:
         - name: tls
           mountPath: "/tmp/k8s-metrics-server/serving-certs/"
           readOnly: true
+        livenessProbe:
+          httpGet:
+            path: /healthz
```

Comment:
Does the kube-rbac-proxy server expose this endpoint (at the specified port)

Reply:
Not sure about that. How can we check? Backplane?

Reply:
This is the same port as the one for the `readinessProbe`. @thetechnick @ashishmax31 Is it possible for two different containers in the same pod to expose the same endpoints on the same port?

Reply:
Nope you are right @garrettthomaskth , the containers in a pod share the same network namespace, so the port should be different